components/HistoricalWithdrawal.tsx

```diff
@@ -0,0 +1,159 @@
+import { VFC } from "react";
+import { IntrinsicElements } from "@/types";
+import { css } from "@emotion/react";
+import {
+	Theme,
+	Accordion,
+	AccordionSummary,
+	AccordionDetails,
+	TextField,
+	Tooltip,
+} from "@mui/material";
+import ExpandMore from "@mui/icons-material/ExpandMore";
+import HelpOutlineIcon from "@mui/icons-material/HelpOutline";
+
+interface HistoricalWithdrawalProps {
+	historicalEventProofId: number;
+	setHistoricalEventProofId: Function;
```

Comment:
Have a more explicit type for these setState type function, in this case it can be `setHistoricalEventProofId: Dispatch<SetStateAction<number>>` with `number` being the type of `historicalEventProofId`. Check out `BridgeProvider` for more examples